client/state/selectors/is-calypsoify-gutenberg-enabled.js

```diff
@@ -16,7 +15,7 @@ export const isCalypsoifyGutenbergEnabled = ( state, siteId ) => {
 	}
 
 	// We do want Calypsoify flows for Atomic sites
-	if ( isSiteAutomatedTransfer( state, siteId ) ) {
+	if ( isJetpackSite( state, siteId ) ) {
```

Comment:
This will force all Jetpack sites to use Calypsoify wp-admin redirects, effectively overriding Gutenframe flows. 
What we want to achieve is the opposite of that (`isCalypsoifyGutenbergEnabled` should always return `false` regardless of site type). Given that I think we can probably remove this selector along with all of its usages.

Reply:
Ooooh, I see – lemme just `return false;` for this PR, and do a followup PR to remove it all.

Reply:
Sounds good, that's exactly what I did for my local testing. :) We can clean it up afterwards in a separate PR.

Reply:
Indeed, works great with posts, pages, CPTs – sweet!